src/middlewared/middlewared/plugins/rdma/interface/crud.py

```diff
@@ -97,9 +97,12 @@ async def do_delete(self, id_):
         data = await self.get_instance(id_)
 
         # Attempt to remove the live configuration
-        result = await self.middleware.call('rdma.interface.configure_interface', data['node'], data['ifname'], None)
-        if not result:
-            self.logger.warn("Failed to delete active RDMA interface configuration")
+        try:
+            result = await self.middleware.call('rdma.interface.configure_interface', data['node'], data['ifname'], None)
+            if not result:
+                self.logger.warn("Failed to delete active RDMA interface configuration")
+        except Exception:
+            self.logger.error('Failed to remove live RDMA configuration', exc_info=True)
```

Comment:
You can do self.logger.exception to remove the exc_info=True. Not required just fyi